src/views/clusteroverview/SettingsTab/LiveMigrationTab/Limits/Limits.tsx

```diff
@@ -0,0 +1,109 @@
+} from '../utils/utils';
+
+const Limits = ({ hyperConverge }) => {
+  const { t } = useKubevirtTranslation();
+
+  const [migrationPerNode, setMigrationPerNode] = useState<number>();
+  const [migrationPerCluster, setMigrationPerCluster] = useState<number>();
+  const updateValuesCluster = useDebounceCallback(updateLiveMigrationConfig, 500);
+  const updateValuesNode = useDebounceCallback(updateLiveMigrationConfig, 500);
+
+  useEffect(() => {
+    if (hyperConverge) {
+      const liveMigrationConfig = getLiveMigrationConfig(hyperConverge);
+      migrationPerCluster ??
+        setMigrationPerCluster(+liveMigrationConfig?.parallelMigrationsPerCluster || 0);
+      migrationPerCluster ??
+        setMigrationPerNode(+liveMigrationConfig?.parallelOutboundMigrationsPerNode || 0);
+    }
+  }, [hyperConverge, migrationPerCluster, migrationPerNode]);
+
+  return (
+    <>
+      <Text component={TextVariants.small}>
+        {t('Set the migrations limits per cluster and per node')}
+      </Text>
+      <div className="live-migration-tab__number--container">
+        <div className="live-migration-tab__number--cluster">
+          <Title headingLevel="h6" size="md">
+            {t('Max. Migration per cluster')}
+          </Title>
+          {migrationPerCluster ? (
+            <NumberInput
+              value={migrationPerCluster}
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setMigrationPerCluster(() => {
+                  updateValuesCluster(hyperConverge, +event.target.value, MIGRATION_PER_CLUSTER);
+                  return +event.target.value;
+                });
+              }}
+              min={0}
+              onMinus={() =>
+                setMigrationPerCluster((mpc) => {
+                  updateValuesCluster(hyperConverge, mpc - 1, MIGRATION_PER_CLUSTER);
+                  return mpc - 1;
+                })
+              }
+              onPlus={() =>
+                setMigrationPerCluster((mpc) => {
+                  updateValuesCluster(hyperConverge, mpc + 1, MIGRATION_PER_CLUSTER);
+                  return mpc + 1;
+                })
+              }
+              inputName={MIGRATION_PER_CLUSTER}
+            />
+          ) : (
+            <Skeleton width={'140px'} height={'33px'} />
+          )}
+          <div>some text</div>
+        </div>
+        <div>
+          <Title headingLevel="h6" size="md">
+            {t('Max. Migration per node')}
+          </Title>
+          {migrationPerNode ? (
+            <NumberInput
+              value={migrationPerNode}
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                setMigrationPerNode(() => {
+                  updateValuesNode(hyperConverge, +event.target.value, MIGRATION_PER_NODE);
+                  return +event.target.value;
+                })
+              }
+              min={0}
+              onMinus={() =>
+                setMigrationPerNode((mpn) => {
```

Comment:
can you rename `mpn` to a more readable var?

Reply:
It's actually a callback value inside a state setter called setMigrationPerNode so I call the arg mpn :) but sure I can change it